server/routes/bookings/bookingsController.test.ts

```diff
@@ -75,4 +75,19 @@ describe('Bookings homepage', () => {
         expect($('h2').text()).not.toContain('How to change your booking')
       })
   })
+
+  it('should render the bookings home page - if no prisoner was present on initial call', () => {
+    sessionData.booker.prisoners = undefined
+    visitService.getFuturePublicVisits.mockResolvedValue([orchestrationVisitDto])
+    bookerService.getPrisoners.mockResolvedValue([prisoner])
```

Comment:
Probably no longer need this test now that the behaviour of the route has changed (or at least `bookerService` mock can go as it's no longer relevant)